api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -574,7 +565,7 @@ public static Module getModuleByPackage(String modulePackage) {
 	 * @see Daemon#startModule(Module)
 	 */
 	public static Module startModule(Module module) throws ModuleException {
-		return Daemon.startModule(module);
```

Comment:
Yes. But the problem with your change is that the module no longer starts in a daemon thread.